contracts/CampaignFactory.sol

```diff
@@ -49,7 +57,16 @@ contract Campaign {
     }
 
     function receiveFund() public payable {
+        require(msg.value >= minimumPayment);
+
+        contributers.push(msg.sender);
```

Comment:
hi @yujhenchuang412 
I think we also need to check next:
- contributor is not the owner of campaign (owner cannot send money to his own campaign)
- contributor can send money many times I think, not sure. We can check if contributor is already in the list of contributor and don't increase contributors/approvers number in this case

Reply:
@Iratar You are correct and I was trying to modify it yesterday.

1. I will add a require() statement to add the restriction.
2. I verified this on Remix., and yes, a contributor can send money multiple times with the same address. So I think I'm going to use a mapping(address ==> some_struct) to record contributors and their funding info including: (1) bool (isApprover) and (2) minFundOfThisContributor to distinguish if they are eligible to be an approver. and just so a contributor can contribute multiple times.